OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -596,8 +596,8 @@ bool AircraftCanEnter(Actor a)
 			if (self.AppearsHostileTo(a))
 				return false;
 
-			var canRearmAtActor = rearmable != null && rearmable.Info.RearmActors.Contains(a.Info.Name);
-			var canRepairAtActor = repairable != null && repairable.Info.RepairActors.Contains(a.Info.Name);
+			var canRearmAtActor = rearmable != null && rearmable.CanRearmAt(a);
```

Comment:
return early if true, avoiding a call to `CanRepairAt`.

Reply:
ok